sites/partners/src/components/applications/PaperApplicationForm/PaperApplicationForm.tsx

```diff
@@ -163,18 +163,16 @@ const ApplicationForm = ({ listingId, editMode, application }: ApplicationFormPr
   return (
     <LoadingOverlay isLoading={loading}>
       <>
-        <StatusBar
-          tagStyle={
-            application?.status == ApplicationStatus.submitted
-              ? AppearanceStyleType.success
-              : AppearanceStyleType.primary
-          }
-          tagLabel={
-            application?.status
+        <StatusBar>
+          <Tag
+            variant={application?.status == ApplicationStatus.submitted ? "success" : "primary"}
```

Comment:
Would it make sense to create a new Variant enum for the Seeds components?

Reply:
If I'm following (lmk if this isn't what you mean) I think that variant should be an enum in seeds (that we should change [this line over there](https://github.com/bloom-housing/ui-seeds/blob/b5bb785a807824640356b0329b3cff2be2351191/src/text/Tag.tsx#L9))

Reply:
Oh I think that could be helpful! My main concern was just leaving us open to mistyping variants or if there are any variants that were carried over from UI-C but aren't in seeds. But with the linked code, I'm realizing now that it would still throw a type error! I don't have very strong feelings on the seeds enum and appreciate the context.